packages/editor-ui/src/components/executions/global/GlobalExecutionsList.vue

```diff
@@ -344,7 +352,9 @@ async function onAutoRefreshToggle(value: boolean) {
 						:key="execution.id"
 						:execution="execution"
 						:workflow-name="getExecutionWorkflowName(execution)"
+						:workflow-permissions="getExecutionWorkflowPermissions(execution)"
```

Comment:
We should avoid calling functions in the template. Could the permissions for all rendered executions get cached in a key-value computed prop? (with execution id as key, permissions as value)